test/services/actionRulesService.test.js

```diff
@@ -0,0 +1,37 @@
+const actionRulesService = require('../../server/services/actionRulesService')
+const actionsService = require('../../server/services/actionsService')
+const actionID = 'FG1'
+const ruleID1 = 3
+const ruleID2 = 5
+const enabled = false
+
+describe('actionRulesService', () => {
```

Comment:
The `actionRulesService` depends on `rulesService`. Ideally, that should be mocked for these unit tests. The `rulesService.updateRuleEnabled` function should be tested separately alongside the existing `rulesService.get` tests in `/test/services/rulesService.test.js`.

Reply:
Yes, good point. I have moved the tests to the `rulesService.test.js` as the complexity is in the `rulesService` not the `actionRulesService`.

Reply:
Yes, agreed (misread your first comment suggesting this), been done now.